SYCL/ESIMD/api/bin_and_cmp_ops_heavy.cpp

```diff
@@ -282,13 +284,15 @@ int main(void) {
   passed &= test<unsigned char, int, 1, BinOp, VSf, IDf>(div_op, q);
   passed &= test<char, float, 7, BinOp, VEf, IDf>(div_op, q, 0.000001f);
 #ifndef WA_BUG
-  passed &= test<short, double, 7, BinOp, VSf, IDf>(div_op, q);
+  if (dev.has(aspect::fp64))
+    passed &= test<short, double, 7, BinOp, VSf, IDf>(div_op, q);
 #endif // WA_BUG
   passed &= test<float, float, 32, BinOp, VEf, IDf>(div_op, q, 0.000001f);
   passed &= test<half, char, 1, BinOp, verify_n, IDf>(div_op, q, 1);
   passed &= test<half, unsigned int, 32, BinOp, VSf, IDf>(div_op, q, 1);
```

Comment:
Would you please add check for fp16?

Reply:
I'd rather do it in a different PR. Fp64 is way more urgent because our CI is broken.